cmd/dependabot/internal/cmd/update.go

```diff
@@ -61,22 +61,25 @@ func NewUpdateCommand() *cobra.Command {
 			}
 
 			if err := infra.Run(infra.RunParams{
-				CacheDir:      cache,
-				Creds:         input.Credentials,
-				Debug:         debugging,
-				Expected:      nil, // update subcommand doesn't use expectations
-				ExtraHosts:    extraHosts,
-				InputName:     file,
-				Job:           &input.Job,
-				LocalDir:      local,
-				Output:        output,
-				ProxyCertPath: proxyCertPath,
-				ProxyImage:    proxyImage,
-				PullImages:    pullImages,
-				Timeout:       timeout,
-				UpdaterImage:  updaterImage,
-				Writer:        writer,
-				Volumes:       volumes,
+				CacheDir:            cache,
+				Creds:               input.Credentials,
+				Debug:               debugging,
+				EnableOpenTelemetry: enableOpenTelemetry,
+				Expected:            nil, // update subcommand doesn't use expectations
+				ExtraHosts:          extraHosts,
+				InputName:           file,
+				Job:                 &input.Job,
+				LocalDir:            local,
+				Output:              output,
+				ProxyCertPath:       proxyCertPath,
+				ProxyImage:          proxyImage,
+				CollectorConfigPath: collectorConfigPath,
```

Comment:
nit: looks like the rest of these parameters are ordered alphabetically.

Reply:
No, it's inherited from the root command, like the other images.